app/javascript/components/enrollment/Enrollment.js

```diff
@@ -111,6 +113,16 @@ class Enrollment extends React.Component {
     if (data.patient.symptom_onset !== undefined && data.patient.symptom_onset !== null && data.patient.symptom_onset !== this.props.patient.symptom_onset) {
       data.patient.user_defined_symptom_onset = true;
     }
+    if (this.state.enrollmentState.first_positive_lab) {
+      if (this.props.edit_mode) {
+        let diffKeysLab = Object.keys(this.state.enrollmentState.first_positive_lab).filter(
+          k => _.get(this.state.enrollmentState.first_positive_lab, k) !== _.get(this.props.first_positive_lab, k)
+        );
+        data['laboratory'] = { id: this.props.first_positive_lab.id, ..._.pick(this.state.enrollmentState.first_positive_lab, diffKeysLab) };
```

Comment:
I think I found a bug where `first_positive_lab` ends up being null here when it shouldn't be. Steps:
1. Enroll a new case with a Symptom Onset Date
2. On the monitoree details page, click Edit on Case Information
3. Check the No Symptom History checkbox and enter a lab result
4. Click next

At this point a I get mostly blank page

![image](https://user-images.githubusercontent.com/8235974/113076779-d0912c00-919d-11eb-9815-55690bbe0cf8.png)

And here's the error in the console

![image](https://user-images.githubusercontent.com/8235974/113076964-282f9780-919e-11eb-9297-9dc918a94329.png)

Reply:
good catch, fixed here: https://github.com/SaraAlert/SaraAlert/pull/747/commits/e2c9d22524075aca7ef71f466ed78527bb6f324d#diff-4db480ec24a44308ae9e46fd4ccfc147b86f1809d19ce73a86a78fab4d51d5d3R117